feat(instance): add section to cover stream attach support (#51)
canonical/anbox-cloud-docs: howto/instance/create.md

See [Provided images](https://discourse.ubuntu.com/t/provided-images/24185) for a list of images that are available in Anbox Cloud.

### Launch an instance with streaming enabled

> *since 1.23.0*

Reviewer: The more we all start to contribute doc updates in advance, the more I feel like we should have separate branches for each release version. But that's a separate conversation. Let's discuss it further.

> *since 1.23.0*

Reviewer: This should be 1.22.0

Reviewer: Corrected it while resolving conflicts.

> If you want to stream the visual output from an instance you have to specify this at creation time. If not being specified the instance cannot be streamed. In previous versions of Anbox Cloud it was only possible to create such an instance via the [Stream Gateway API](https://canonical.github.io/anbox-cloud.github.com/latest/anbox-stream-gateway/). The same can now be achieve through the [AMS API](https://canonical.github.io/anbox-cloud.github.com/latest/ams/) or the `amc` CLI.

Reviewer 2: ```suggestion
If you want to stream the visual output from an instance you have to specify this at creation time. If not being specified the instance cannot be streamed. In previous versions of Anbox Cloud it was only possible to create such an instance via the [Stream Gateway API](https://canonical.github.io/anbox-cloud.github.com/latest/anbox-stream-gateway/). The same can now be achieved through the [AMS API](https://canonical.github.io/anbox-cloud.github.com/latest/ams/) or the `amc` CLI.
```

> 

Reviewer 3: Is it worth mentioning how the agent API token is created? Another question is who will be in charge of creating the agent API token? charm or end user?

Author: It will all be automatically setup for the user in all cases (microbox, appliance or charms)

Reviewer 3: Alright, that's the missing piece for now. 
We should create a story to track it.

Reviewer 3: https://warthogs.atlassian.net/browse/AC-2415

> To enable streaming for a new instance simply run

Reviewer: ```suggestion
To enable streaming for a new instance, simply run:
```

    amc launch --enable-streaming ...

AMS will automatically create a streaming session for the instance. You can find the id of the session as a tag on the instance in the format `session=<id>`.

> If you want to to further customize the display settings or frame rate you can use the `--display-size`, `--display-density` and `--fps` arguments. To create an instance with a 1080p resolution, a frame rate of 60 and a DPI of 120 you can run

Reviewer 2: ```suggestion
If you want to further customise the display settings or frame rate you can use the `--display-size`, `--display-density` and `--fps` arguments. To create an instance with a 1080p resolution, a frame rate of 60 and a DPI of 120 you can run
```

> If you want to to further customize the display settings or frame rate you can use the `--display-size`, `--display-density` and `--fps` arguments. To create an instance with a 1080p resolution, a frame rate of 60 and a DPI of 120 you can run

Reviewer: I feel this reads better, WDYT?

```suggestion
If you want to to further customise the streaming configuration such as display settings or frame rate, use the corresponding arguments: `--display-size`, `--display-density` and `--fps`. For example, to create an instance with a 1080p resolution, a frame rate of 60 and a DPI of 120, run:
```
